int32 index
uint64 time_stamp
uint8[1048576] data

uint32 MAX_SIZE = 1048576
